src/data/fintype/card.lean

```diff
@@ -96,11 +96,6 @@ finset.prod_insert_none f univ
 
 end
 
-@[to_additive]
-theorem fin.prod_univ_def [comm_monoid β] {n : ℕ} (f : fin n → β) :
-  ∏ i, f i = ((list.fin_range n).map f).prod :=
-by simp [fin.univ_def, finset.fin_range]
-
 @[to_additive]
 theorem finset.prod_range [comm_monoid β] {n : ℕ} (f : ℕ → β) :
```

Comment:
I think this should move too

Reply:
It is already moved!

Reply:
Or did you mean moving also `finset.prod_range`?

Reply:
Yes, I meant `finset.prod_range` as that's the line this comment is on!

Reply:
Ok, thanks for the review. First, I will try to fix the broken imports until mathlib builds, and then I will do this move.